src/main/java/com/parasoft/report/transformer/XMLToSarif.java

```diff
@@ -118,4 +136,25 @@ private void convertXmlToSarif() {
         }
         Logger.info(MessageFormat.format("SARIF report has been created: {0}", this.outputSarifReport.getAbsolutePath()));
     }
+
+    private boolean isAbsolutePath(String path) {
+        Pattern pattern = Pattern.compile(ABSOLUTE_PATH_REGEX);
+        return pattern.matcher(path).matches();
+    }
+
+    private void avoidDuplicateProjectRootPaths(String[] paths) {
+        Set<String> uniquePaths = new HashSet<>();
+        for (String path : paths) {
+            path = path.endsWith("/") ? path : path + "/";
+            if (uniquePaths.contains(path)) {
+                throw new IllegalArgumentException(MessageFormat.format("Duplicate project root path found: {0}", path));
```

Comment:
Should we just ignore duplicates but print out warning?

Reply:
Absolutely, I agree with you.